docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -456,9 +456,29 @@ In the NHANES example, the full first sentence of the above quote was:
 3.2.1 Global Burden of Disease (GBD)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. todo::
-  Describe the best methodology/terminology for working with the "sex" variable from GBD
+The GBD study is very complex and many different data sources. Here, 
+we provide a general overview of how the demographics team works with the "sex" 
+variable and some of the limitations. To understand "sex" in the context of a 
+particular disease or risk factor, please reference the `GBD methods appendix <https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(20)30925-9/fulltext#supplementaryMaterial>`_ to 
+see the exact data sources used. 
+
+The high level steps used for calculating population estimates for each 
+country, year, age group and sex are: 
+
+  #. First, GBD uses fertility rate estimates from hospital records and surveys such as DHS to define the population. A standard sex ratio at birth is applied to all. 
+  #. The population is then aged up from birth, with mortality. In the 0-5 age group, mortality estimates come from DHS surveys, vital registries and death records. 
```

Comment:
For each of these data sources, it would be good to at least nod at how they might measure sex/gender. For example I'm guessing surveys use an undifferentiated report from a parent/guardian, and death records may depend on a medical professional's assessment. (I'm not sure exactly what "vital registries" here would mean, besides death records.)

That way we're showing how the conflation builds up through the GBD modeling process.

Reply:
Add some examples as discussed!